### DISCUSSION TITLE
Validator Tool Billing_Code_Type

### DISCUSSION THREAD

**Discussion by tysonpkp** [CMSgov/price-transparency-guide-validator#50] [Ideas]
Does there need to be any type of validation for Billing Code Type being populated with a valid value?  

We've started generating QA files and found that after running through the validator tool, it will pass the validation, but upon manual review we've found places our logic has inadvertently excluded a billing code type.

Here is a snippet from our JSON:

    "negotiation_arrangement": "ffs",
    "name": "PHYSICAL THERAPY/VISIT",
    "billing_code_type": "",  <------------- See the lack of value in the billing code type.
    "billing_code_type_version": "2022",
    "billing_code": "421",

**Comment by shaselton-usds** [comment]
@tysonpkp I am going to move this Discussion over to the [validator repo](https://github.com/CMSgov/price-transparency-guide-validator),

Thanks.
